### PR TITLE
Add security settings with good defaults

### PR DESCRIPTION
**Pull request by mythmon**
This should make all our problems go away in dev's `/__heartbeat__`.

@Osmose r?

### CODE REVIEW

File: normandy/settings.py [hunk 1]
```diff
@@ -122,6 +122,15 @@ class Base(Core):
     ]
     USE_X_FORWARDED_HOST = values.BooleanValue(False)
     SECURE_PROXY_SSL_HEADER = values.TupleValue()
+    SECURE_HSTS_SECONDS = values.IntegerValue(1)
```
Comment (Osmose) [inline normandy/settings.py]: Should probably be an hour.

File: normandy/settings.py [hunk 2]
```diff
@@ -122,6 +122,15 @@ class Base(Core):
...
+    SECURE_HSTS_INCLUDE_SUBDOMAINS = values.BooleanValue(True)
+    CSRF_COOKIE_HTTPONLY = values.BooleanValue(True)
+    CSRF_COOKIE_SECURE = values.BooleanValue(True)
+    SECURE_SSL_REDIRECT = values.BooleanValue(True)
+    SESSION_COOKIE_SECURE = values.BooleanValue(True)
+    SECURE_BROWSER_XSS_FILTER = values.BooleanValue(True)
+    SECURE_CONTENT_TYPE_NOSNIFF = values.BooleanValue(True)
+    X_FRAME_OPTIONS = values.Value('DENY')
```
Comment (Osmose) [inline normandy/settings.py]: Does the self-repair view work with this?

**Comment by Osmose** [comment]
r+wc, nice work!

File: normandy/settings.py (hunk 2 above)
Comment (mythmon) [inline normandy/settings.py]: Yes

**Pull request merged**